src/components/EpisodePreview/EpisodePreview.jsx

```diff
@@ -0,0 +1,128 @@
+import React, { useContext } from "react";
+import {
+	EpisodesContext as Episodes
+} from "COMPONENTS/Episodes";
+
+export default function EpisodePreview(props){
+
+	// CONTEXT
+	//-----------------------------
+	const episodes = useContext(Episodes);
+
+
+	// RENDER
+	// ----------------------------
+	const { 
+		episodeId,
+		index,
+		episodeNo,
+		title,
+		thumbnail: {
+			src: thumbSrc,
+			description: thumbDesc
+		},
+		trailer: {
+			src: trailerSrc
+		}
+	} = props;
+
+	// handle visibility of active episode and episode preview overlay
+	const isActiveEpisode 	= episodes.get.activeIndex === index;
+	const thisOverlayActive = isActiveEpisode && episodes.get.overlayActive;
+	const hidden 			= !thisOverlayActive;
+
+	const isFirst 	= index === 0;
+	const isLast 	= index === episodes.get.episodeCount - 1;
+
+	const prevEpisodeNo = episodeNo - 1;
+	const nextEpisodeNo = episodeNo + 1;
+
+	const previewId = title
+		.replace(/\W+/g, '-') // remove non-alphanumeric & replace with -s
+		.toLowerCase()
+		+"-preview"
+		;
```

Comment:
You're doing two things here:
1. making it url-safe
2. appending 'preview'. 

I would say that a better way to do this would be to change the `generateSafeString` in the utils so that you can optionally pass in a second argument to determine whether you use a `_` or a `-` to replace words, and then do:

```
const previewId = generateSafeString(`${title} preview`, "-");
```
or
```
const previewId = `${generateSafeString(title, "-")}-preview`;
```